Spanner/tests/Unit/V1/SpannerClientTest.php

```diff
@@ -230,8 +211,7 @@ public function commitTest()
         $transport->addResponse($expectedResponse);
         // Mock request
         $formattedSession = $gapicClient->sessionName('[PROJECT]', '[INSTANCE]', '[DATABASE]', '[SESSION]');
-        $mutation = new \Google\Cloud\Spanner\V1\Mutation();
-        $response = $gapicClient->commit($formattedSession, [$mutation]);
+        $response = $gapicClient->commit($formattedSession);
```

Comment:
The error in the tests comes from here, because `$mutation` is removed

Reply:
It's interesting because `SpannerClient::commit` requires `$mutations`, but I can't find out why. Looking at [spanner.proto](https://github.com/googleapis/googleapis/blob/master/google/spanner/v1/spanner.proto#L1005), it doesn't seem to be marked required. However, the mutation message does have [a field which is marked required](https://github.com/googleapis/googleapis/blob/master/google/spanner/v1/mutation.proto#L39), so could this be causing it?

Still, I'm not sure what would have changed this behavior just for the tests.